c7n/commands.py

```diff
@@ -171,6 +171,27 @@ def _print_no_policies_warning(options, policies):
         log.warning('Empty policy file(s).  Nothing to do.')
 
 
+class DuplicateKeyCheckLoader(SafeLoader):
+
+    def construct_mapping(self, node, deep=False):
```

Comment:
Wondering if different naming convention would make sense - `preprocess_mapping` or something, since it really is just returning a `CSafeLoader` unless there is an exception?

Reply:
this is hooking a base class method, renaming the method isn't a valid option.